lms/envs/common.py

```diff
@@ -1247,7 +1247,8 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
```

Comment:
@clrux The modernizr in the platform is just the copy from the Pattern Library. I believe it is the full version rather than a limited subset, but I don't know for sure. 

It would be great to get rid of these checked in files, and instead replace them with the Pattern Library versions. Maybe that should be a separate pull request.